src/main/kotlin/app/revanced/patches/all/screenshot/noblock/patch/NoScreenshotBlockPatch.kt

```diff
@@ -0,0 +1,111 @@
+package app.revanced.patches.all.screenshot.noblock.patch
+
+import app.revanced.extensions.findMutableMethodOf
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.replaceInstruction
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.util.proxy.mutableTypes.MutableMethod
+import org.jf.dexlib2.Opcode
+import org.jf.dexlib2.iface.instruction.formats.Instruction35c
+import org.jf.dexlib2.iface.reference.MethodReference
+
+@Patch(false)
+@Name("no-screenshot-block")
```

Comment:
Please reflect this change to the package and class name as well
```suggestion
@Name("remove-screenshot-restriction")
```

Reply:
Would `app/revanced/patches/all/screenshot/removerestriction/patch/RemoveScreenshotRestrictionPatch.kt` be okay?

Reply:
Looks fine.